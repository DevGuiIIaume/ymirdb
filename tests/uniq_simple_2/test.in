uniq unique
set unique 11 11 23 11 11 0 1 1
get unique
uniq unique
get unique
del unique
uniq unique
bye
