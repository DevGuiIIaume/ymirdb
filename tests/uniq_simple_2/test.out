> no such key

> ok

> [11 11 23 11 11 0 1 1]

> ok

> [11 23 11 0 1]

> ok

> no such key

> bye
